src/query-compiler/default-query-compiler.ts

```diff
@@ -938,6 +938,15 @@ export class DefaultQueryCompiler
   ): void {
     this.visitNode(node.name)
     this.append(' as ')
+
+    if (isBoolean(node.materialized)) {
+      if (node.materialized) {
+        this.append('materialized ')
+      } else {
+        this.append('not materialized ')
+      }
```

Comment:
alternatively could...

```suggestion
      if (!node.materialized) {
        this.append('not ')
      }
      
      this.append('materialized ')
```

I try to avoid `else` as much as possible.